WhateverGreen/kern_igfx.cpp

```diff
@@ -996,11 +996,55 @@ OSObject *IGFX::wrapCopyExistingServices(OSDictionary *matching, IOOptionBits in
 	return FunctionCast(wrapCopyExistingServices, callbackIGFX->orgCopyExistingServices)(matching, inState, options);
 }
 
+bool IGFX::removeVTSupportedProfile2(IOService *that) {
+	if (!that)
```

Comment:
Makes no sense to check this, as it is dereferenced right before calling this function.

Reply:
https://github.com/acidanthera/WhateverGreen/commit/f9cff542db12c3257068490039a2b8535bc016be